lib/api.js

```diff
@@ -1476,6 +1476,26 @@ module.exports = function(self, options) {
     }
   };
 
+  self.revert = function(req, commitId, callback) {
```

Comment:
Near as I can figure this method is going to resolve with `(null, doc)` where doc is the updated draft, because of how waterfall behaves. OK but then later when you call find() you're passing `result.id`, not `result._id`.